tests/ci/cdk/cdk/codebuild/github_ci_linux_arm_omnibus.yaml

```diff
@@ -262,6 +262,29 @@ batch:
         variables:
           AWS_LC_CI_TARGET: "tests/ci/run_fips_tests.sh"
 
+    - identifier: amazonlinux2_clang7x_aarch_fips_callback
+      buildspec: ./tests/ci/codebuild/common/run_simple_target.yml
+      env:
+        type: ARM_CONTAINER
+        privileged-mode: false
+        compute-type: BUILD_GENERAL1_LARGE
```

Comment:
Would medium be sufficient here? seems like these tests finish fast

Reply:
Turns out we use large for all the FIPS targets because medium runs out of memory running the ACVP tests.